citation.cff

```diff
@@ -0,0 +1,35 @@
+cff-version: 1.2.0
+message: "If you use this lesson material, please cite it using these metadata."
+authors:
+- name: "CodeRefinery"
+- family-names: "Bast"
+  given-names: "Radovan"
+- family-names: "Wikfeldt"
+  given-names: "Kjartan Thor"
+- family-names: "Rantaharju"
+  given-names: "Jarno"
+- family-names: "Darst"
+  given-names: "Richard"
+- family-names: "Wittke"
+  given-names: "Samantha"
+- family-names: "van der Burg"
+  given-names: "Sven"
+- family-names: "Razick"
+  given-names: "Sabry"
+- family-names: "Rasel"
+  given-names: "Annajiat Alim"
+- family-names: "Negru"
+  given-names: "Stefan"
+- family-names: "Lindi"
+  given-names: "Bjørn"
+- family-names: "van Vliet"
+  given-names: "Marijn"
+- family-names: "Orozco"
+  given-names: "Luisa"
+title: "CodeRefinery Documentation lesson"
```

Comment:
I am wondering if we can come up with a better title than "CodeRefinery Documentation lesson". Thinking ...

Reply:
Maybe "CodeRefinery lesson: Documenting code" or "Documenting research software (CodeRefinery lesson)"?

Reply:
Good point! 
Maybe something like "Code documentation for researchers" or "How to document your research software" ?

Reply:
Ah it did not show me your second comment while writing. Also good! 
Probably good to still have CodeRefinery in the title :thinking:

Reply:
I like "How to document your research software" a lot.